test_plans/sub_group_mask.asciidoc

```diff
@@ -16,33 +16,106 @@ is selected on the CTS command line.
 
```

Comment:
I noticed that this link above is broken;

> This is a test plan for the APIs described in
https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/GroupMask/GroupMask.asciidoc[GroupMask.asciidoc]

Can you fix the link and also change the text of the link to be "sycl_ext_oneapi_sub_group_mask" (which is the name of the extension)?